### ISSUE TITLE
500 server error on nwc2musicxml.appspot.com 

### ISSUE THREAD

**Issue by janoks** [lasconic/nwc2musicxml#36]
It seems like nwc2musicxml.appspot.com got issues. I get server error 500 when I try to load the page. Any chance this can be fixed? I love the converter and use it a lot, as I can write scores in NWC much faster than in Musescore.

**Comment by lasconic** [comment]
Here is a way to run the tool in this repository. Hope it helps.


* Install git and clone the project if you know how to do it
or download the project and unzip : https://github.com/musescore/nwc2musicxml/archive/refs/heads/master.zip

* Install Java and Ant.

* In the project directory, run the following commands

```
ant package
java -cp build/NwcTxt2MusicXML.jar fr.lasconic.nwc2musicxml.convert.Nwc2MusicXML TestFiles/TestVoltas.nwctxt test.xml
```

You can also convert all the files in a directory with

```
java -cp build/NwcTxt2MusicXML.jar fr.lasconic.nwc2musicxml.convert.batch.BatchConvert  ADIRECTORYFULLOFNWCFILE
```

**Comment by alanp30** [comment]
will the above work on a mac?

**Comment by lasconic** [comment]
Yes. I actually tested only on a mac but it should work anywhere.

**Comment by alanp30** [comment]
Thanks @lasconic Are you able to help me more? I'm a complete novice with github. Can't quite work out what to do! Thanks

**Comment by dmland** [comment]
As of 2022-09-26 on MacOSX Catalina, I had to change lines 5 & 6 to "1.7" to get a successful build … which I did. Thank you for making this so easy.
